### PR TITLE
Outlook.com added

### PR DESCRIPTION
**Pull request by glacasa**

### CODE REVIEW

File: _data/main.yml [hunk 1]
```diff
@@ -99,7 +99,21 @@ sections:
           goog: Yes
           authy: Yes
           sms: Yes
-          doc: https://support.google.com/accounts/answer/180744?hl=en
+          doc: http://windows.microsoft.com/en-us/windows/two-step-verification-faq
```
Comment (computmaxer) [inline _data/main.yml]: This shouldn't have been changed, it is the documentation for GMail's two factor.

File: _data/main.yml [hunk 2]
```diff
@@ -99,7 +99,21 @@ sections:
...
+
+  - id: email
+    title: Email
+    icon: mail
```
Comment (computmaxer) [inline _data/main.yml]: Looks like the Email section was duplicated here. Checkout how multiple sites are listed in other sections in this file, and add your outlook section below GMail in that fashion. Thanks!

**Comment by glacasa** [comment]
Yes, sorry, I made my modifications too fast.
- I think it's ok now

**Comment by computmaxer** [comment]
Thanks for the correction  :+1:

**Pull request merged**